src/UmbracoExamine.PDF/UmbracoExamine.PDF.csproj

```diff
@@ -1,321 +1,39 @@
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="app.config" />
-    <None Include="packages.config" />
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="..\packages\Umbraco.SqlServerCE.4.0.0.1\build\Umbraco.SqlServerCE.targets" Condition="Exists('..\packages\Umbraco.SqlServerCE.4.0.0.1\build\Umbraco.SqlServerCE.targets')" />
-  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
+        <TargetFramework>net5.0</TargetFramework>
+        <ContentTargetFolders>.</ContentTargetFolders>
+        <Version>9.0.0</Version>
+        <AssemblyVersion>9.0.0</AssemblyVersion>
+        <InformationalVersion>9.0.0-beta001</InformationalVersion>
+        <FileVersion>9.0.0</FileVersion>
+        <Authors>Umbraco HQ</Authors>
+        <Company>Umbraco CMS</Company>
+        <Copyright>Copyright © Umbraco 2021</Copyright>
+        <PackageProjectUrl>http://umbraco.com/</PackageProjectUrl>
+        <PackageLicenseUrl>http://opensource.org/licenses/MIT</PackageLicenseUrl>
+        <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+        <Product>UmbracoExamine.PDF</Product>
+        <PackageId>UmbracoExamine.PDF</PackageId>
+        <Title>UmbracoExamine.PDF</Title>
+        <Description>Umbraco indexing for PDF files using UmbracoExamine</Description>
+        <NeutralLanguage>en-US</NeutralLanguage>
+        <PackageTags>Umbraco Examine Lucene PDF UmbracoExamine</PackageTags>
+        <PackageIconUrl>https://umbraco.com/dist/nuget/logo-small.png</PackageIconUrl>
+        <RepositoryType>git</RepositoryType>
+        <RepositoryUrl>https://github.com/umbraco/UmbracoExamine.PDF</RepositoryUrl>
     </PropertyGroup>
-    <Error Condition="!Exists('..\packages\Umbraco.SqlServerCE.4.0.0.1\build\Umbraco.SqlServerCE.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Umbraco.SqlServerCE.4.0.0.1\build\Umbraco.SqlServerCE.targets'))" />
-  </Target>
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
-</Project>
\ No newline at end of file
+
+    <ItemGroup>
+        <PackageReference Include="PdfPig" Version="0.1.4" />
+        <PackageReference Include="PDFsharpNetStandard2" Version="1.51.4845" />
+        <PackageReference Include="Umbraco.Cms.Web.Website" Version="9.0.0-preview20210607.65117" />
+        <PackageReference Include="Umbraco.Cms.Web.BackOffice" Version="9.0.0-preview20210607.65117" />
```

Comment:
If we are not using anything since last beta, I think we should just use beta3, then we can ship a beta1 of this :)

Reply:
Just had a look at it, sadly we use `LuceneDirectoryFactory` and the `LuceneIndexDiagnostics` which seems to also have changed since beta003.

Reply:
Cool, then we just wait shipping this to next beta